lib/commands/connect/shared.js

```diff
@@ -182,6 +182,19 @@ exports.withConnection = function(context, heroku) {
   });
 }
 
+var withMapping = exports.withMapping = function(connection, object_name) {
+  return Q.Promise(function(resolve, reject) {
+    var found = false, object_name_lower = object_name.toLowerCase();
+    connection.mappings.forEach(function(mapping) {
+      if (mapping.object_name.toLowerCase().indexOf(object_name_lower) == 0) {
```

Comment:
I wonder if we should allow guid as well as names?  More of a maybe later than immediate concern.